prime-dotnet-webapi/Extensions/ClaimsPrincipalExensions.cs

```diff
@@ -2,6 +2,8 @@
 using System.Linq;
 using System.Security.Claims;
 
+using Prime.Models;
+
 namespace Prime
 {
     public static class ClaimsPrincipalExensions
```

Comment:
Is this supposed to be Extensions?

Reply:
Need Prime.Models because CanAccess has Enrollee as a parameter